imports/ui/components/detailLeft.jade

```diff
@@ -0,0 +1,9 @@
+template(name='detailLeft')
```

Comment:
I agree with you on having small templates in the same file being a good thing. It's a more common pattern now to put templates into separated files, due to .js/.coffee files becoming bulky and hard to read when they contain Template helpers/events/hooks for different templates.